.cascade-code/Chapman.edu/_cascade/formats/modular/mastheads/side_panel_hero.vm

```diff
@@ -0,0 +1,129 @@
+#*
+ * side_panel_hero(.vm)
+ * Master Version: Chapman.edu/_cascade/formats/modular/mastheads/side_panel_hero
+ *#
+
+#import( "/_cascade/formats/helpers.velocity" )
+
+#macro(outputSidePanelHero $element)
+    ## Panel Variables
+    #set ( $panelTitle    = $element.getChild('panel').getChild('title').value )
+    #set ( $panelSubtitle = $element.getChild('panel').getChild('subtitle').value )
+    #set ( $panelBody     = $_SerializerTool.serialize($element.getChild('panel').getChild('body'), true ) )
+    #set ( $dropdownLabel = $element.getChild('panel').getChild('dropdownNav').getChild('dropdownLabel').value )
+    #set ( $dropdownLinks = $_XPathTool.selectNodes($element, '//sidePanelHero/panel/dropdownNav/dropdownLink') )
+    #set ( $buttons       = $_XPathTool.selectNodes($element, '//sidePanelHero/panel/callsToAction/button') )
+
+    ## Featured Content Slides
+    #set ( $slides = $_XPathTool.selectNodes($element, '//sidePanelHero/featuredContent') )
+
+    <div class="side-panel-hero">
+
+      <div class="hero-content">
+        <section class="hero-content-panel">
+        	<h1>$_EscapeTool.xml($panelTitle)</h1>
+			#if ($panelSubtitle != "")
+    		    <h2>$_EscapeTool.xml($panelSubtitle)</h2>
+            #end
```

Comment:
This file has some very weird indentation going on, like here, and around line 49, 90. Is this file using 2-space tabs? And am I the only one seeing the spacing thing

Reply:
It does have formatting issues. It's probably a mix of tabs and spaces. I didn't fix it because, well I was paying close attention to it and I'm lazy and I hate redeploying Cascade files. 

I'll fix it.